src/App.tsx

```diff
@@ -1,30 +1,30 @@
-import Footer  from './components/Footer';
-import PrimaryNav from './components/PrimaryNav';
-import PostsWrap from './components/PostsWrap';
-import { initiatePostHog } from './lib/analytics';
-import { useState } from 'react';
-import { BrowserRouter } from 'react-router-dom';
-import RepoSubmission from './components/RepoSubmission';
-import GradBackground from './components/GradBackground';
+import Footer from "./components/Footer";
```

Comment:
You can revert this file, because it is hard to see what your changes are

Reply:
I think this file, came from the beta branch I merged with my branch. I didn't work on this at all